apps/api/src/app/environments-v1/usecases/output-renderers/maily-to-liquid/maily.types.ts

```diff
@@ -16,6 +16,7 @@ export enum MailyAttrsEnum {
   SHOW_IF_KEY = 'showIfKey',
   EACH_KEY = 'each',
   FALLBACK = 'fallback',
+  ALIAS_FOR = 'aliasFor',
```

Comment:
Is ALIAS_FOR working as new TipTap node type or a node attribute?

Reply:
Its only an attribute providing more info about the variable.

We will very likely need more of those in the future since knowing only string variable name is very limited.